ff/src/fields/models/quadratic_extension.rs

```diff
@@ -89,34 +89,6 @@ pub trait QuadExtParameters: 'static + Send + Sync + Sized {
     /// A specializable method for multiplying an element of the base field by
     /// the appropriate Frobenius coefficient.
     fn mul_base_field_by_frob_coeff(fe: &mut Self::BaseField, power: usize);
-
-    /// A specializable method for exponentiating that is to be used
-    /// *only* when `fe` is known to be in the cyclotommic subgroup.
-    fn cyclotomic_exp(fe: &QuadExtField<Self>, exponent: impl AsRef<[u64]>) -> QuadExtField<Self> {
```

Comment:
There's no need for this to be in the `QuadExtParameters` trait anymore, which simplifies things slightly.